File: Lista 01 - Java /Ex03/src/ex03/Ex03.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package ex03;

import java.util.Scanner;

/**
 * @author bryan
 */

public class Ex03 {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        System.out.print("Digite o salário do funcionário: ");
        float salario = sc.nextInt();

        float aumento = 0;

        if (salario <= 1200) {
            aumento = salario * 0.1f;
        } else if (salario > 1200) {
            aumento = salario * 0.05f;
        }

        salario += aumento;

        System.out.println("O funcionario teve um aumento de: R$ " + aumento + "\nE seu novo salário é: R$ " + salario);
    }
}

File: Prática/Lista 04 - C (Introdução)/Ex03/Ex03.c
#include "../Console.c";
#include <stdio.h>
#include <stdlib.h>

int main()
{
    clear();
    float salario;
    int aumento;

    printf("Digite o salário do funcionário: ");
    scanf("%f", &salario);  

    if(salario <= 1200) {
        aumento = 10;
        salario += salario*0.1;
    }else{
        aumento = 5;
        salario += salario*0.05;
    }

    printf("\nO funcionário teve um aumento de %d%% passando a R$ %.2f\n\n",aumento, salario);

    return 0;
}



